sphinx_gallery/gen_rst.py

```diff
@@ -124,7 +125,7 @@ def flush(self):
 """
 
 
-CODE_OUTPUT = """.. rst-class:: sphx-glr-script-out
+CODE_OUTPUT = u""".. rst-class:: sphx-glr-script-out
```

Comment:
Did you try putting it in while on this branch, or on a previous one? It might work okay with this one since the unicode reading is a bit more unified.